### PR TITLE
Slight simplification and log cleanup

### PR DESCRIPTION
**Pull request by EdPutans**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,6 +122,22 @@ A behaviour handler is a function that can either:
...
+
+From a functionality perspective Triggers and Handlers are the same, but use different methods of invoking a behaviour.
+
+an `APITrigger` is a behaviour controlled by the backend. When we receive a supported string for that (e.g. `'BEHAVIOUR_MODAL'`), we need to search through `clientTriggers` to find one that matches, then invoke the attached `invoke` method.
+
+a `Handler` is custom behaviour handled by the DiFi-provider wrapped app, and is controlled directly via `invoke`
+
+TLDR in pseudo:
+```ts
+type APITrigger = { ...mutual,  behaviour: 'Supported behaviour' }
```
Comment (AdenFraser) [inline README.md:135]: I'm not convinced by the 'APITrigger' from a language perspective 🤔

Not sure why, but it doesn't feel right. Longer term all triggers will be setup and configured by the backend as our 'machine' should be intelligent enough to know when to fire them, even custom ones. So I can't see any instance where a Trigger isn't invoked by the B/E.

**Comment by AdenFraser** [comment]
I don't think the seperation between 'Trigger' and 'Handler' is sensible. 'Handler' is a hangover from before we had product language, and actually Trigger and Handler represent the same thing -> to create a behaviour change for a visitor.

If the concern here is to support custom trigger behaviours, such as that on the Booking Flow Frontend, we should re-align the Booking Flow with the direction of Fingerprint, not create added complexity to Fingerprint this early on.

File: src/context/CollectorContext.tsx [hunk 2]
```diff
@@ -85,20 +87,48 @@ export function CollectorProvider({
     }
   }, [intently, log])
 
+  const invokeAPITrigger = (locatedTrigger: APITrigger) => {
```
Comment (AdenFraser) [inline src/context/CollectorContext.tsx:90]: I don't understand the need for any of this logic I'm afraid

File: src/utils/typeHelpers.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { APITrigger, Trigger } from '../client/types'
+
+// determine whether a trigger object is an APITrigger instance.
+// use this to distinguish between APITriggers and manual Handlers
+export function isTriggerAnApiTrigger(t: Trigger): t is APITrigger {
```
Comment (AdenFraser) [inline src/utils/typeHelpers.ts:5]: And same for this one, unfortunately

**Comment by EdPutans** [comment]
This PR now only contains the tiny life improvements

**Pull request closed without merging**